### PR TITLE
chore(VRT Pipelines): remove unneeded variables from yml files

### LINKED ISSUE #27010: Remove shouldBuildstorybookaddon from template for VR Tests pipeline
With changes in #27006, this is no longer necessary and can be removed.

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Changes:
- Removes `shouldBuildStorybookAddon` and `shouldBuildNorthstar` variables from .yml files as the changes in #27006 makes them unnecessary. 

## Related Issue(s)

- Fixes #27010

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/Avatar-mount_base.html">1584</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/Avatar-mount.html">1550</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/Button-mount_base.html">1148</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/Button-mount.html">1125</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/Field-mount_base.html">2417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/Field-mount.html">2387</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/FluentProvider-mount_base.html">1961</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/FluentProvider-mount.html">1917</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/FluentProviderWithTheme-mount_base.html">732</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/FluentProviderWithTheme-mount.html">740</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">695</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">697</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">734</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/InfoButton-mount_base.html">666</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/InfoButton-mount.html">657</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/MakeStyles-mount_base.html">2227</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/MakeStyles-mount.html">2202</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/Persona-mount_base.html">3517</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/Persona-mount.html">3435</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/SpinButton-mount_base.html">2842</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test-react-components/SpinButton-mount.html">2802</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: .devops/templates/runpublishvrscreenshot.yml [hunk 1]
```diff
@@ -8,12 +8,6 @@ parameters:
   - name: vrTestPackagePath
```
Comment (fabricteam) [inline .devops/templates/runpublishvrscreenshot.yml:8]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27040&commitId=619d8ee5f87103e530fd834c0f739b3689279c42&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

File: .devops/templates/runpublishvrscreenshot.yml (hunk 1 above)
Comment (fabricteam) [inline .devops/templates/runpublishvrscreenshot.yml:8]: ## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27040&commitId=619d8ee5f87103e530fd834c0f739b3689279c42&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Inverted.chromium.png|246| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/BaseButton-mount_base.html">1372</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/BaseButton-mount.html">1400</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Breadcrumb-mount_base.html">3462</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Breadcrumb-mount.html">3396</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Checkbox-mount_base.html">3058</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Checkbox-mount.html">3053</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/CheckboxBase-mount_base.html">2754</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/CheckboxBase-mount.html">2723</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ChoiceGroup-mount_base.html">5109</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ChoiceGroup-mount.html">5173</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ComboBox-mount_base.html">1474</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ComboBox-mount.html">1459</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/CommandBar-mount_base.html">11101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/CommandBar-mount.html">11110</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ContextualMenu-mount_base.html">16053</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ContextualMenu-mount.html">15748</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DefaultButton-mount_base.html">1605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DefaultButton-mount.html">1616</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DetailsRow-mount_base.html">3970</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DetailsRow-mount.html">4036</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DetailsRowFast-mount_base.html">4034</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DetailsRowFast-mount.html">4087</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DetailsRowNoStyles-mount_base.html">3892</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DetailsRowNoStyles-mount.html">3865</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Dialog-mount_base.html">3634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Dialog-mount.html">3640</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DocumentCardTitle-mount_base.html">660</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/DocumentCardTitle-mount.html">658</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Dropdown-mount_base.html">3740</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Dropdown-mount.html">3749</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/FocusTrapZone-mount_base.html">2335</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/FocusTrapZone-mount.html">2356</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/FocusZone-mount_base.html">2298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/FocusZone-mount.html">2294</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedList-mount_base.html">59293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedList-mount.html">65425</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedList-virtual-rerender_base.html">28069</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedList-virtual-rerender.html">28182</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">105574</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedList-virtual-rerender-with-unmount.html">105544</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedListV2-mount_base.html">640</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedListV2-mount.html">655</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedListV2-virtual-rerender_base.html">622</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedListV2-virtual-rerender.html">646</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">659</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">651</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/IconButton-mount_base.html">2220</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/IconButton-mount.html">2217</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Label-mount_base.html">860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Label-mount.html">849</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Layer-mount_base.html">5025</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Layer-mount.html">5015</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Link-mount_base.html">953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Link-mount.html">977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/MenuButton-mount_base.html">1928</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/MenuButton-mount.html">1927</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/MessageBar-mount_base.html">2670</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/MessageBar-mount.html">2749</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Nav-mount_base.html">3827</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Nav-mount.html">3813</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/OverflowSet-mount_base.html">1570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/OverflowSet-mount.html">1561</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Panel-mount_base.html">2962</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Panel-mount.html">2937</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Persona-mount_base.html">1532</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Persona-mount.html">1486</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Pivot-mount_base.html">1908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Pivot-mount.html">1930</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/PrimaryButton-mount_base.html">1776</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/PrimaryButton-mount.html">1800</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Rating-mount_base.html">8085</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Rating-mount.html">8181</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/SearchBox-mount_base.html">1771</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/SearchBox-mount.html">1783</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Shimmer-mount_base.html">3276</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Shimmer-mount.html">3311</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Slider-mount_base.html">2480</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Slider-mount.html">2433</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/SpinButton-mount_base.html">5414</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/SpinButton-mount.html">5430</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Spinner-mount_base.html">929</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Spinner-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/SplitButton-mount_base.html">3646</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/SplitButton-mount.html">3570</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Stack-mount_base.html">962</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Stack-mount.html">952</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/StackWithIntrinsicChildren-mount_base.html">2647</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/StackWithIntrinsicChildren-mount.html">2644</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/StackWithTextChildren-mount_base.html">5495</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/StackWithTextChildren-mount.html">5495</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/SwatchColorPicker-mount_base.html">11993</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/SwatchColorPicker-mount.html">11992</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/TagPicker-mount_base.html">3015</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/TagPicker-mount.html">3008</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/TeachingBubble-mount_base.html">95655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/TeachingBubble-mount.html">96249</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Text-mount_base.html">925</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Text-mount.html">915</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/TextField-mount_base.html">1831</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/TextField-mount.html">1818</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ThemeProvider-mount_base.html">1752</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ThemeProvider-mount.html">1770</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ThemeProvider-virtual-rerender_base.html">1264</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ThemeProvider-virtual-rerender.html">1243</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2515</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2501</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Toggle-mount_base.html">1316</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/Toggle-mount.html">1280</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/buttonNative-mount_base.html">636</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27040/perf-test/buttonNative-mount.html">636</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by ling1726 — APPROVED** [review]

**Pull request merged**